1주차/[Lv.2]_[1차]_캐쉬/최원석.js

```diff
@@ -0,0 +1,33 @@
+function solution(cacheSize, cities) {
+  let hit = 0;
+  const caches = [];
+  let arr = [];
+  cities = cities.map((v) => v.toUpperCase());
+
+  for (let i = 0; i < cities.length; i++) {
+    arr.push(cities[i]);
```

Comment:
원석님 코드에서 루프를 돌리는 부분을
같은 횟수만큼 루프가 돌아간다면, 묶어서 실행하면 좋을 것 같습니다!
```suggestion
for (let i = 0; i < cities.length; i++) {
    arr.push(cities[i].toUpperCase());
```